src/app/app-routing.module.ts

```diff
@@ -1,10 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { HomeComponent } from './components/home/home.component';
+import { ServiceDetailComponent } from './components/service-detail/service-detail.component';
 
 const routes: Routes = [
+  { path:'servicedetail', component: ServiceDetailComponent },
```

Comment:
aca el path hay que ponerlo con un guion service-detail para tenerlo en kebab-case https://wesleygrimes.com/angular/2019/02/24/angular-routing-best-practices-for-enterprise-applications.html

Reply:
Listo